settings.xml

```diff
@@ -1,93 +1,17 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<settings xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd" xmlns="http://maven.apache.org/SETTINGS/1.1.0"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+                      http://maven.apache.org/xsd/settings-1.0.0.xsd">
+
   <servers>
     <server>
-      <username>${artifactory-username}</username>
-      <password>${artifactory-password}</password>
-      <id>artifactory</id>
-    </server>
-    <server>
```

Comment:
By removing that one you made maven angry. If I do:
`rm -rf ~/.m2 && mvn --settings settings.xml -Dartifactory-username=****** -Dartifactory-password=**** -Dzimbra.version=8.8.15 clean install` it gives me the following error:
```
[ERROR] Failed to execute goal on project open-zal: Could not resolve dependencies for project com.zextras:open-zal:jar:2.23.0-SNAPSHOT: Failed to collect dependencies at com.zextras:gen-build-info:jar:0.0.1-SNAPSHOT: Failed to read artifact descriptor for com.zextras:gen-build-info:jar:0.0.1-SNAPSHOT: Could not transfer artifact com.zextras:gen-build-info:pom:0.0.1-SNAPSHOT from/to the-lucky-gunslingers (https://zextras.jfrog.io/artifactory/the-lucky-gunslingers-java-sdk): authentication failed for https://zextras.jfrog.io/artifactory/the-lucky-gunslingers-java-sdk/com/zextras/gen-build-info/0.0.1-SNAPSHOT/gen-build-info-0.0.1-SNAPSHOT.pom, status: 401 Unauthorized -> [Help 1]
```

I'd reinstate the deleted lines to the-luky-gunslingers repo.

Reply:
I've added suggestions here and there to solve that issue. If I did not forgot any of them we should be fine :ok_hand: